static/webpack.config.js

```diff
@@ -118,13 +116,6 @@ const config = {
   },
   module: {
     rules: [
-      {
-        test: /\.(js|jsx|ts|tsx)$/,
-        exclude: /node_modules/,
```

Comment:
do we not need this rule anymore?

Reply:
nope, looks like babel is not needed anymore